hist/histdraw/v7/inc/ROOT/RHistDrawable.hxx

```diff
@@ -83,71 +83,18 @@ public:
 
 };
 
-extern template class RHistDrawable<1>;
-extern template class RHistDrawable<2>;
-extern template class RHistDrawable<3>;
-
```

Comment:
I have no idea why this code was here. 
@Axel-Naumann probably has better idea.
All other changes are ok for me.

Reply:
OK, thanks @linev! I'll merge it once the builds are OK, and we'll see what @Axel-Naumann says...